Client.xcodeproj/project.pbxproj

```diff
@@ -4118,7 +4118,7 @@
 					};
 					F84B225B1A09210A00AAB793 = {
 						CreatedOnToolsVersion = 6.1;
-						DevelopmentTeam = 9G8J6YA743;
+						DevelopmentTeam = 43AQ936H96;
```

Comment:
I'm not sure why Xcode keeps changing this but I'm pretty sure we don't want the teams changing. Maybe it doesn't like the fact that we use two different teams based on scheme?